fetch.bs

```diff
@@ -3093,16 +3093,36 @@ steps:
 <i>authentication-fetch flag</i>.
 
 <ol>
+ <li><p>Let <var>httpRequest</var> be a new <a for=/>request</a>.
+
+ <li><p>If <var>request</var>'s <a for=request>window</a> is "<code>no-window</code>" and
+ <var>request</var>'s <a for=request>redirect mode</a> is "<code>error</code>", then set
+ <var>httpRequest</var> to <var>request</var>.
+
  <li>
-  <p>Let <var>httpRequest</var> be <var>request</var> if
-  <var>request</var>'s <a for=request>window</a> is "<code>no-window</code>"
-  and <var>request</var>'s <a for=request>redirect mode</a> is
-  "<code>error</code>", and the result of <a lt=clone for=request>cloning</a>
-  <var>request</var> otherwise.
+  <p>Otherwise, if <var>request</var>'s <a for=request>body</a> is null, then set
+  <var>httpRequest</var> to a copy of <var>request</var> except for its <a for=request>body</a>.
```

Comment:
I'm not sure. The fact that the note needs to be duplicated makes me think it's not.